spec/controllers/income_employment_status_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe IncomeEmploymentStatusController do
+  let(:step) { assigns(:step) }
+  let(:member) do
+    create(:member, sex: "male", relationship: "Child", ssn: "12345")
+  end
+  let(:invalid_params) do
+    {
+      step: { members: { member.id => { employment_status: "" } } },
```

Comment:
Thought about breaking this up to multiple lines? Nested hashes, at least to me, are easier to scan when expanded out